shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -57,11 +57,11 @@
 </template>
 
 <template name="_loginButtonsLoggedOutDropdown">
-  <h4>Sign in</h4>
-
   {{#if isDemoUser}}
-    <p class="demo-note">Sign in to make your demo data permanent.</p>
+    <h4>Create Account</h4>
```

Comment:
https://github.com/sandstorm-io/sandstorm/wiki/Shell-Design-Guidelines from Nena says "Capitalization: - always mixed caps with the first letter capitalized"